docs/bundles/SyliusGridBundle/configuration.rst

```diff
@@ -33,7 +33,37 @@ Here you will find all configuration options of ``sylius_grid``.
                         enabled: true
                         template: ~
                         position: 100
-                        options: { }
+                        options:
+                            fields: { }
+                        form_options: { }
```

Comment:
To show that it is an array? I wanted to be consistent with the rest of the file.